### ISSUE TITLE
Replace comma-separated envvars with comma+whitespace separated envvars

### ISSUE THREAD

**Issue by mrnoname1000** [itzg/docker-minecraft-server#2682]
### Enhancement Type

Improve an existing feature

### Describe the enhancement

Seems like there are some envvars that split on commas, some on commas+newlines, some on commas+whitespace, and others on just whitespace. I propose replacing all of the first type with comma+whitespace splits unless they rely on newlines being retained (which I don't see being an issue). In places where commas are required (e.g. in [mc-image-helper](https://github.com/itzg/mc-image-helper#mcopy), and are they really required?), values can first be split on commas+whitespace, then re-joined on commas:
```sh
IFS=,$IFS                          # prepend comma to $IFS
plugins=($PLUGINS)                 # split on $IFS (comma+whitespace)
mc-image-helper mcopy ... "$*"     # join with comma
IFS=${IFS#?}                       # restore $IFS
```

The motivation behind this issue is this snippet from my docker-compose.yml:
```yaml
SPIGET_RESOURCES:
  # spark,discordsrv,deathchest,pluginmanager,spark,veinminer #,vault,essentialsx
  57242,18494,101066,69061,57242,12038 #,34315,9089
```
Notice how I accidentally duplicated one of the plugins. Ignoring how cryptic YAML multiline indicators are, I would prefer to write this instead:
```yaml
SPIGET_RESOURCES: >-
  57242   # spark
  18494   # discordsrv
  101066  # deathchest
  69061   # pluginmanager
  12038   # veinminer
  #34315   # vault
  #9089    # essentialsx
```

I can work on a PR for this if we're in agreement!

**Comment by itzg** [comment]
Yes, those need to be brought into consistency of commas or newlines. Spaces might be significant so it is harder to declare that as the ubiquitous arg delimiter. There is a constant already for comma or newline 

https://github.com/itzg/mc-image-helper/blob/388da11f847afd663ccd08f7544cb0720e1de000/src/main/java/me/itzg/helpers/McImageHelper.java#L95

A PR would be great, but change only one place first so I can see what you have in mind.

**Comment by itzg** [comment]
...actually `SPIGET_RESOURCES` is an unusual case since that's one of the next things I want to port over to mc-image-helper. At that point, it'll get the proper comma or whitespace/newline arg delimiting support. Also, support for inline comments is actually a feature that [I was discussing with someone on Discord](https://discord.com/channels/660567679458869252/1209712955906719854/1209862036843003934).
